app/Observers/AttendanceObserver.php

```diff
@@ -5,13 +5,19 @@
 namespace App\Observers;
 
 use App\Attendance;
+use App\Jobs\CreateUserFromBuzzAPI;
 use App\Jobs\PushToJedi;
 
 class AttendanceObserver
 {
     public function saved(Attendance $attendance): void
     {
         if (null === $attendance->attendee) {
+            // I know this will not cause a PushToJedi run, but if the user is being created from attendance they will
+            // not have access to anything with Jedi anyway.
+            CreateUserFromBuzzAPI::dispatch(CreateUserFromBuzzAPI::IDENTIFIER_GTID, $attendance->gtid)
+                ->onQueue('buzzapi');
```

Comment:
I would like to pull major information at this point. You have two options:
 - `CreateUserFromBuzzAPI` always dispatches a major update (in which case you should probably just combine the jobs)
 - Allow the major job to accept a GTID so that you can directly chain the jobs here.